quickstarts/azure/azure-virtual-machines/config.yml

```diff
@@ -0,0 +1,28 @@
+name: azure-virtual-machines
+description: |-
+  ## What is Azure Virtual Machines?
+
+  A service for creating and managing a virtual or simulated computing environment as an alternative to a physical environment.
+
+  ### Get started!
+
+  Start monitoring Azure Virtual Machines by connecting Microsoft Azure to New Relic!
+
+  Check out our Azure Virtual Machines documentation to instrument your cloud service and manage the stability, scalability, and reliability of your systems with New Relic's infrastructure monitoring capabilities.
+summary: |-
```

Comment:
```suggestion
summary: |
```

Reply:
the schema compliance check is failing, this may be causing it.

Reply:
I just had to add "description": null to the json file.